assets/js/vendor/index.js

```diff
@@ -3,3 +3,12 @@
 require('nodep-date-input-polyfill');
 
 require('jquery.mmenu');
+
+require('tooltipster');
+
+$(document).ready(function() {
+  $('.hastip').tooltipster({
+     trigger: 'click',
```

Comment:
Also, move it to https://github.com/opentrials/opentrials/blob/master/assets/js/index.js instead. This file should only contain the `requires`.